ds701_book/20-Recommender-Systems.qmd

```diff
@@ -81,40 +79,43 @@ Some examples:
 * Priority inbox & spam filtering (Google)
 * Online dating (OK Cupid)
 
+---
+
 A more formal view:
     
 * User - requests content
 * Objects - that can be displayed
```

Comment:
I feel this description of objects is a bit too generic. Can we describe these as the pool of relevant instances of content?